pkg/shell/machines/machines.js

```diff
@@ -127,7 +132,7 @@ function Machines() {
         overlay: {
             localhost: {
                 visible: true,
-                manifests: cockpit.manifests
+                manifests: import_manifests(cockpit.manifests)
```

Comment:
I'm curious -- how many ts errors disappear with this? (do a before/after comparison with ts-relaxed?)

Reply:
None. :-)

From TypeScripts point of view, manifests always were fully typed. What is different now is that we don't crash at run-time in random places with a invalid manifest anymore, but just output a validation error.